src/js/common/schemaform/widgets/CurrencyWidget.jsx

```diff
@@ -22,7 +22,7 @@ export default class CurrencyWidget extends React.Component {
       this.props.onChange();
     } else {
       // Needs to look like a currency
-      if (!val.match(/^\${0,1}[0-9,]*(\.\d{1,2})?$/)) {
+      if (!/^\${0,1}[0-9,]*(\.\d{1,2})?$/.test(val)) {
```

Comment:
I would imagine so; it can probably fail fast where `val.match` can't (maybe?) and doesn't have to find all the parentheses matches.

I'm not committed enough to make a benchmark right now, though. :man_shrugging: